var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -40,10 +41,18 @@ class Vtk(CMakePackage):
     # At the moment, we cannot build with both osmesa and qt, but as of
     # VTK 8.1, that should change
     conflicts('+osmesa', when='+qt')
+    conflicts('+python', when='+python3')
+    conflicts('+python3', when='@:8.0')
```

Comment:
Since the version range is inclusive, not exclusive, this should be changed to `@:7` so it's available in `>=8` instead of strictly `>8.0`

Reply:
My intention was to move to python3 only 8.1.1 and 8.1.2, so indeed 8.0.1 would be python2 only. So this was by design. But we can certainly do the move starting from 8.

Reply:
Okay, it''s fine then as is, I misunderstood the intent.  I get confused by the version specs sometimes and forget that `8.0` actually means `8.0.x` and not `8.0.0`